Service/OS/AndroidGCMNotification.php

```diff
@@ -57,14 +65,16 @@ class AndroidGCMNotification implements OSNotificationServiceInterface
     /**
      * Constructor
      *
-     * @param $apiKey
+     * @param bool         $dryRun
+     * @param string       $apiKey
      * @param bool         $useMultiCurl
      * @param int          $timeout
      * @param Logger       $logger
-     * @param AbstractCurl $client       (optional)
+     * @param AbstractCurl $client (optional)
      */
-    public function __construct($apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
+    public function __construct($dryRun, $apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
```

Comment:
I'd prefer this at the end of the signature if possible - with a default to `false` (after the `$client` parameter)